Analysis/bin/CreateSpectralHists.cxx

```diff
@@ -126,9 +130,9 @@ class CreateSpectralHists {
     void AddTau(const Tau& tau)
     {
         if(PassSelection(tau)) {
-            const auto gen_match = static_cast<analysis::GenLeptonMatch>(tau.lepton_gen_match);
+            const auto gen_match = static_cast<analysis::GenLeptonMatch>(tau.genLepton_kind);
```

Comment:
Type of `genLepton_kind` is `reco_tau::gen_truth::GenLepton:: Kind`. Don't cast it to `analysis::GenLeptonMatch`, because in the future these definitions can diverge.